### PR TITLE
23 Create icons of the app

### PR DESCRIPTION
**Pull request by MaasboDev**
### :tophat: Goal
Use the icons that were created for the design system in Figma, using the right icon for debug or release and both themes, light and dark mode and adding a splash screen using them.

### :memo: Type of Change
_Choose the option that best describes the type of change you are making_
- [ ] Bugfix (fixing an issue)
- [X] New feature (adding functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] Refactoring (changing the code structure without adding or removing functionality)
- [ ] Chore (update dependencies, build config... and other non-feature code changes)

### :scroll: Checklist
_Make sure your PR meets the following points before requesting a review_
- [X] I have followed the project's code style guidelines
- [X] I have self-reviewed my code
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
-

### CODE REVIEW

**Comment by soygabimoreno** [comment]
I would add something to the README to avoid this issue.
image

In [Meditation](https://github.com/soygabimoreno/Meditation) we have this:
image

**Comment by soygabimoreno** [comment]
Do you need help with something in particular in this PR, @MaasboDev?

**Comment by soygabimoreno** [comment]
Also, something for this:
image

**Comment by soygabimoreno** [comment]
> Do you need help with something in particular in this PR, @MaasboDev?

I assume it's this:
image


It would be cool to put a comment here in the PR also at the point where it's failing for easy understanding

File: app/build.gradle.kts [hunk 1]
```diff
@@ -77,7 +77,7 @@ android {
 }
 
 dependencies {
-    implementation(project(":core:design-system"))
+    implementation(projects.core.designSystem)
```
Comment (soygabimoreno) [inline app/build.gradle.kts:80]: Is this on purpose?

I see this on `master`.
image

**Comment by MaasboDev** [comment]
> I would add something to the README to avoid this issue. image
> 
> In [Meditation](https://github.com/soygabimoreno/Meditation) we have this: image

Oops, sorry I forgot to share this with you, but now that this project is public, I'm a bit scared to share this information publicly.

**Comment by soygabimoreno** [comment]
> Oops, sorry I forgot to share this with you, but now that this project is public, I'm a bit scared to share this information publicly.

They should be fake data of course

**Comment by MaasboDev** [comment]
Remaking the module fixed the problem.

**Review by soygabimoreno — APPROVED** [review]

**Pull request merged**